### PR TITLE
Add obtainium badge

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,8 +31,9 @@
 
 ## 📲 Installation
 
-<a href="https://apt.izzysoft.de/fdroid/index/apk/com.meenbeese.chronos"><img height="80" alt="Izzy Download" src="./assets/badges/izzy_install_badge.png"></a>
-<a href="https://github.com/meenbeese/Chronos/releases/latest"><img height="80" alt="APK Download" src="./assets/badges/apk_install_badge.png"></a>
+<a href="https://apt.izzysoft.de/fdroid/index/apk/com.meenbeese.chronos"><img height="80" alt="Izzy Download" src="./assets/badges/izzy_install_badge.png" align="center"></a>
+<a href="https://github.com/meenbeese/Chronos/releases/latest"><img height="80" alt="APK Download" src="./assets/badges/apk_install_badge.png" align="center"></a>
+<a href="https://apps.obtainium.imranr.dev/redirect?r=obtainium://add/https://github.com/meenbeese/Chronos"><img height="56" alt="APK Download" src="./assets/badges/badge_obtainium.png" align="center"></a>
```
Comment (meenbeese) [inline README.md:36]:
```suggestion
<a href="https://apps.obtainium.imranr.dev/redirect?r=obtainium://add/https://github.com/meenbeese/Chronos"><img height="55" alt="Obtainium Download" src="./assets/badges/badge_obtainium.png" align="center"></a>
```

**Comment by meenbeese** [comment]
Would it make more sense to merge this after https://github.com/ImranR98/apps.obtainium.imranr.dev/pull/70 is merged or is it unrelated?

**Comment by meenbeese** [comment]
By the way, thanks for suggesting this. I didn't know that Obtainium had a badge and a website even as an Obtainium user.

**Comment by Jean-BaptisteC** [comment]
It's can be merged, your PR on their website, add just ability to see app in their catalog.

**Pull request merged**